### ISSUE TITLE
Errors when running example pcap download/parsing code

### ISSUE THREAD

**Issue by jpjamipark** [timpalpant/go-iex#4]
Thanks for writing this library, it's very helpful for me.

I'm new to Go, and currently trying to run the sample code in the readme. I've created a new directory in `go/src`, and when I try to run the file using `go run testfile.go`, I get the following error:
```
# command-line-arguments
./hello.go:33:19: cannot assign 1 values to 2 variables
./hello.go:33:45: cannot use resp.Body (type io.ReadCloser) as type iex.PacketDataSource in argument to iex.NewPcapScanner:
	io.ReadCloser does not implement iex.PacketDataSource (missing LinkType method)
```

As a Go newbie, I'd appreciate any help with understanding this error. Thanks!

**Comment by timpalpant** [comment]
Sorry about that, the example in the README needs an update to reflect the changes in e9356d7ff0e2bc8a0fe9d6b96082d537cefb6aeb.

Fixed in f7ef89fb5009c5f4b7b3b24f90404d1cda3b3f14, please give it another shot with the latest example.

**Comment by jpjamipark** [comment]
This
